src/lib/controller/message_handler.ts

```diff
@@ -96,7 +98,17 @@ export class ControllerMessageHandler {
         return {};
       }
       case ControllerCommand.removeNodeFromAllAssocations: {
-        await driver.controller.removeNodeFromAllAssocations(message.nodeId);
+        if (client.schemaVersion > 2) {
+          throw new NotSupportedBySchemaError(client.schemaVersion, "<3");
+        }
+        await driver.controller.removeNodeFromAllAssociations(message.nodeId);
+        return {};
+      }
+      case ControllerCommand.removeNodeFromAllAssociations: {
+        if (client.schemaVersion < 3) {
+          throw new NotSupportedBySchemaError(client.schemaVersion, ">2");
+        }
+        await driver.controller.removeNodeFromAllAssociations(message.nodeId);
```

Comment:
Alternatively, we could also just accept both commands and not raise. 
```suggestion
      case ControllerCommand.removeNodeFromAllAssociations: {
        await driver.controller.removeNodeFromAllAssociations(message.nodeId);
```

Reply:
(this suggest won't work because of the curly braces, but something like that.